packages/strapi-plugin-content-manager/admin/src/containers/EditViewDataManagerProvider/index.js

```diff
@@ -196,7 +196,8 @@ const EditViewDataManagerProvider = ({
       inputValue = null;
     }
 
```

Comment:
why remove the biginteger condition ?

Reply:
Because the `biginteger` does not exist on this input and since the `biginteger` is a string I not sure I should reset it to `null`. In fact it was a mistake from my side to add this condition.